crate/operator/grand_central.py

```diff
@@ -192,10 +192,10 @@ def get_grand_central_deployment(
                             ],
                             resources=V1ResourceRequirements(
                                 limits={
-                                    "cpu": 2,
-                                    "memory": "150Mi",
+                                    "cpu": "500m",
+                                    "memory": "256Mi",
                                 },
-                                requests={"cpu": "100m", "memory": "150Mi"},
+                                requests={"cpu": "100m", "memory": "256Mi"},
```

Comment:
```suggestion
                                requests={"cpu": "10m", "memory": "256Mi"},
```

Most containers don't do anything anyway. And those containers are running on 1:4 CPU / RAM VMs mostly (i.e. 64m @ 256Mi)